jsonnet/main.jsonnet

```diff
@@ -80,9 +80,9 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                  // Because we build OpenShift images separately to upstream,
                  // we have to ensure these versions exist before upgrading.
                  openshiftOauthProxy: 'latest',
-                 prometheus: 'v2.15.2',
-                 alertmanager: 'v0.20.0',
-                 nodeExporter: 'v0.18.1',
+                 prometheus: 'v2.19.2',
+                 alertmanager: 'v0.21.0',
+                 nodeExporter: 'v1.0.1',
                  promLabelProxy: 'v0.1.0',
```

Comment:
Not sure if these versions still hold?

Reply:
no and it never did :D

Reply:
did not set that, so didn't update 🤷‍♀️ I meant any other versions other than the ones I set, do we need to update?

Reply:
This should help our users, so I would say yes we should update them.

Reply:
Added to the Release 4.6 checklist #847